requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 -r requirements.txt
 eth-tester[py-evm]==0.1.0-beta.32,<1.0
+py-evm==0.2.0a26
```

Comment:
Raiden is pinned on version `py-evm==0.2.0a32`, if this is required, please also update the `constraints.txt` in the Raiden repo.

Reply:
Actually, nevermind this. I tested again a couple of times in a fresh venv and I did not get the issue anymore. I think I had some package caching problems, because I got the `eth_typing` error a couple of days ago **with** a fresh venv..